config.js

```diff
@@ -11,7 +11,11 @@ export default /** @type {const} */ ({
         computeAsync: window.location.search.includes('&computeAsync'),
     },
 
-    forceClearNonRelevant: window.location.search.includes('&clearNonRelevant'),
+    /**
+     * When set to `true`, non-relevant values will be treated as blank. This behavior
+     * is largely consistent with JavaRosa.
```

Comment:
largely? That sounds pessimistic? 😄

Reply:
If I recall, that wording is a reference to [this](https://github.com/enketo/enketo-core/pull/870/files#diff-1639f2fb93ca42c1a35786766efa8b81527f63010903193dcf6afd8cf685334fR391-R392). 🙃

Reply:
Can you try that link again? I think it might have been on a commit that's gone.

Reply:
I think it's just GitHub failing to expand the diff in the link? [Here it is](https://github.com/enketo/enketo-core/blob/e6c24bd9ed216115a84b4e5c1b9944055540a083/test/spec/calculate.spec.js#L391).